README.md

```diff
@@ -62,3 +62,44 @@ nvm config is saved in `.nvmrc`, set appropriate node version using:
 ```shell
 nvm use
 ```
+
+## Android
+
+An Android package is generated that includes all icons through the `NrkIcons` object, including expressive variants
+
+The latest version is found under the [Publish Android job](https://github.com/nrkno/core-icons/actions/workflows/publish-android.yml) under the `Publish Android` part (look for `Maven published version: <version>`). Add the library as a depdenceny with `implementation "no.nrk.core:icons:<version>"` (or if more convenient in a core module with `api("no.nrk.core:icons:<version>")`)
+
+### Usage
+
+This library makes it easy to use expressive variants of the icons. See also the [demo app](/android/app/src/main/kotlin/no/nrk/core/icons/MainActivity.kt).
```

Comment:
kun expressive variants?

Reply:
A bit poorly written, I can see how it sounds like only expressive variants are included. Maybe this is better?

`This library makes it easy to use NRK icons, as well as automatically changing to expressive variants`?

Reply:
If I've understood the intent, possibly something in the lines of `(...) as well as providing utility to switch between expressive and regular icon variants of the same icon (globally)`?